packages/central-server/src/dataAccessors/upsertAnswers.js

```diff
@@ -22,21 +22,30 @@ export async function upsertAnswers(models, answers, surveyResponseId) {
       question_id: answer.question_id,
       survey_response_id: surveyResponseId,
     };
-
-    if (answer.type === 'Photo') {
+    const s3Client = new S3Client(new S3());
```

Comment:
This is a good chance to move the handling of files and images from meditrak-app-server to central server and re-use that. It looks pretty much the same but will require a bit of refactoring on the meditrak-app-server side. This might be a good task to pair on with @rohan-bes

Reply:
Did you wanna have a pair on this @acdunham? I think there'd be some time next week if you wanted :+1:

Reply:
Sure, sounds good @rohan-bes, I will make a calendar invite :)